docs/api.yml

```diff
@@ -498,7 +498,7 @@ components:
           type: string
         location:
           type: string
-          example: "NL:Amsterdam"
+          example: "NL#Amsterdam"
```

Comment:
I found that I also used `:` instead of `#` in _docs/example_crowdaction.dynamo.json_. Please also change it in that file.